src/software/ai/navigator/path_planner/hrvo/velocity_obstacle.h

```diff
@@ -0,0 +1,60 @@
+#include "software/geom/vector.h"
```

Comment:
reminder to add `#pragma once` to header

Reply:
done